fixtures/app/extensions/product-subscription/src/index.js

```diff
@@ -12,7 +12,7 @@ function App(root, { extensionPoint }) {
     root.createComponent(
       Text,
       {},
-      `It works the ${extensionPoint} extension! APP_URL is: ${process.env.APP_URL}`
+      `It works the ${extensionPoint} extension! API_KEY is: ${process.env.SHOPIFY_API_KEY}`
```

Comment:
I changed this one to `SHOPIFY_API_KEY` because it caused the bundle generated on deploy not to have the variable substituted.